client/src/components/Register/Register.style.tsx

```diff
@@ -0,0 +1,72 @@
+import styled, { css } from "styled-components";
+import { inputStyle } from "../../helpers/theme/snippets/input";
+import { pageStyle } from "../../helpers/theme/snippets/page";
+import { inputLabelStyle } from "../../helpers/theme/snippets/text";
+
+const elementWidth = "200px";
+const boxShadow = css`
```

Comment:
Put this in `shadow` snippets file as `inputShadow`, that's why the file exists.